diff_robot/launch/diff_robot.launch

```diff
@@ -1,10 +1,14 @@
 <launch>
   <arg name="world" default="empty"/>
   <!-- We resume the logic in empty_world.launch, changing only the name of the world to be launched -->
+
+  <env name="GAZEBO_MODEL_PATH" value="$(find diff_robot)/models:$(optenv GAZEBO_MODEL_PATH)"/>
```

Comment:
It's really weird that a the `export` tag is not doing what it's supposed to do.
BTW, if you change the package name of this find, it will work.